pinot-core/src/main/java/com/linkedin/pinot/core/startree/hll/ArrayConverter.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (C) 2014-2016 LinkedIn Corp. (pinot-core@linkedin.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.pinot.core.startree.hll;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.nio.IntBuffer;
+
+public class ArrayConverter {
+    public static final ByteOrder ENDIAN_ORDER = ByteOrder.BIG_ENDIAN;
+
+    /**
+     * assume each consecutive 4 bytes to be translated to an int
+     * @param byteArray
+     * @return
+     */
+    public static int[] bytesToInts(byte[] byteArray) {
```

Comment:
Nicely done.
I'll suggest naming the method itself as bigEndianBytesToInts(). That will alert the caller about the endian-ness issue and is very clear as to what order is used.

With static final ENDIAN_ORDER he can not change that right now.